docs/examples/gnoi/gnoicode.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: default
+title: "gNOI in Golang"
+date: 2022-01-06 10:07:00 --0600
+categories:
+---
+## gNOI review
+
+These are a few examples leveraging golang and gNOI using the gNOI protobuf specifically the [System service](https://github.com/openconfig/gnoi/blob/master/system/system.proto)
+
+The following examples show how to leverage the [ping rpc](https://github.com/openconfig/gnoi/blob/master/system/system.proto#L41) and [traceroute rpc](https://github.com/openconfig/gnoi/blob/master/system/system.proto#L47)
+
+All code can be found within the [src directory under gnoi](https://github.com/aristanetworks/openmgmt/tree/main/src/gnoi)
+
+We will be leveraging the gNOI [godocs]((https://pkg.go.dev/github.com/openconfig/gnoi)) which module can be imported as github.com/openconfig/gnoi
+
+Each one of the examples has the following constants which can be changed based on the environment.
+
+```golang
+const (
+	username    = "admin"
+	password    = "admin"
+	target      = "172.20.20.2:6030"
+	destination = "1.1.1.1"
+	timeOut     = 5
+)
```

Comment:
can we just change these to command line args in the code?  then folks don't have to build them uniquely.

Reply:
Added flags for each file.

Reply:
Added flags to each.